packages/calcite-components/src/components/list-item/list-item.e2e.ts

```diff
@@ -275,4 +275,20 @@ describe("calcite-list-item", () => {
 
     expect(calciteListItemClose).toHaveReceivedEventTimes(1);
   });
+
+  it("should fire calciteListItemToggle event when opened", async () => {
```

Comment:
Can you update the test to assert on the event emitting for both opening and closing?